documentation/slate/source/includes/objects.md

```diff
@@ -264,6 +264,10 @@ A container containing either a [admin](#admin), [poi](#poi), [address](#address
 
 ### Trip
 
+A trip corresponds to a scheduled train circulation (and all its linked real-time informations).
+
+An example : the train doing Paris-Marseille every day at 19h30 is the Trip named 456789.
+
```

Comment:
Added note about vehicle_journeys here https://github.com/CanalTP/navitia/pull/1650/commits/8f36ece31e10d874b0be50c49751f9aae0c76747 as there is no link in `links` yet